### ISSUE TITLE
Install .net4.5 or update in 4.8?

### ISSUE THREAD

**Issue by FreeM1ne** [UnamSanctam/SilentCryptoMiner#2001]
When I start visual studio it asks me to switch to .net 4.8 or install 4.5, but no matter how many times I try to install 4.5 I can't get it installed. Is there any solution to install .net 4.5?

**Comment by FreeM1ne** [comment]
the first method helped me, which cryptocurrency is now more profitable to mine on a video card? rvn or etc

**Comment by FreeM1ne** [comment]
Is it possible to pass multiple pools separated by commas?

**Comment by UnamSanctam** [comment]
No (it is technically possible if you use the "Advanced Parameters" specific to xmrig or ethminer for multiple pools through the command line) but you can add a pool (or pools) if you use the "Remote Configuration" or web panel configuration, it will store the different pools in this order:
1) Web panel pool configuration
2) "Remote Configuration" pool configuration
3) Builder pool configuration

**Comment by FreeM1ne** [comment]
configs can delete, domain can be blocked, so if possible could you explain how I can do this through advanced settings?

**Comment by UnamSanctam** [comment]
Not sure, for xmrig I think you will have to add everything like this: `--url firstpool --user firstwallet` and all other options that you would use (like SSL), the pools entered into "Advanced Parameters" will come before the pool you entered into the builder. For ethminer it might be something like adding the pool like such: `--pool=stratum://wallet@pool:port`

**Comment by UnamSanctam** [comment]
Yes you can add any amount of "Remote Configurations" as a backup. The character limit for the entire command line is 32767 characters (since it uses the ShellInfo part of the PEB and not actually the command line), so you should be fine no matter how many you add.

**Comment by FreeM1ne** [comment]
And why is there an extra space after "--algo=rx/0 --url=" in the miner settings?
